src/actions/analytics.ts

```diff
@@ -206,6 +210,10 @@ export function AnalyticsActions<ActionsBase extends Constructor<TinlakeParams>>
       return maxRedeem
     }
 
+    getSeniorTokenPrice = async () => {
```

Comment:
i would rename to `calcSeniorTokenPrice` just to differentiate  more from the function below

Reply:
calcSeniorTokenPrice()?